src/app/products/product-card-rating/product-card-rating.component.ts

```diff
@@ -11,4 +11,4 @@ export class ProductCardRatingComponent {
   public stars: number[] = [1, 2, 3, 4, 5];
   public isStartLessOrEqualToRating(_star: number): boolean { 
      return _star <= this.product.rating; }
```

Comment:
Why you use "_star" instead of just "star"?

Reply:
when use 'star' - get error